src/cascadia/TerminalSettingsEditor/SettingContainer.cpp

```diff
@@ -122,23 +122,42 @@ namespace winrt::Microsoft::Terminal::Settings::Editor::implementation
 
         _UpdateOverrideSystem();
 
-        if (const auto& content{ Content() })
+        if (const auto& child{ GetTemplateChild(L"Expander") })
+        {
+            if (const auto& expander{ child.try_as<Microsoft::UI::Xaml::Controls::Expander>() })
+            {
+                // apply header text as name (automation property)
+                if (const auto& header{ Header() })
+                {
+                    if (const auto headerText{ header.try_as<hstring>() })
+                    {
+                        Automation::AutomationProperties::SetName(expander, *headerText);
+                    }
+                }
+
+                // apply help text as tooltip and full description (automation property)
+                if (const auto& helpText{ HelpText() }; !helpText.empty())
+                {
+                    Controls::ToolTipService::SetToolTip(expander, box_value(helpText));
+                    Automation::AutomationProperties::SetFullDescription(expander, helpText);
+                }
+            }
+        }
+        else if (const auto& content{ Content() })

```

Comment:
i actually think, by Carlos' suggestion, you can simplify even more. After all, the only difference between lines 134,141 and 155,162 is nothing. There's no difference...! so you can simplify the code significantly

Reply:
Sounds good, added a helper